docs/known-risks-and-limitations.md

```diff
@@ -0,0 +1,86 @@
+# Dual Governance: Known Risks & Limitations
+
+While the [Dual Governance (DG) system](specification.md) is designed to protect stETH holders from malicious DAO actions, certain protocol specifics and implementation details can reduce its effectiveness in certain situations. This document outlines key risks, limitations, and potential mitigation strategies.
+
+
+## 1. Malicious Oracle Sets
+
+DG relies heavily on the Lido protocol’s [withdrawals mechanics](https://docs.lido.fi/guides/lido-tokens-integration-guide#withdrawals-unsteth), which in turn depend on the performance of the [accounting](https://docs.lido.fi/guides/oracle-spec/accounting-oracle) and [validators exit bus](https://docs.lido.fi/guides/oracle-spec/validator-exit-bus) oracles.
+
+A malicious set of oracle members could completely halt the withdrawal process. Combined with an ongoing RageQuit, this scenario could lock user funds in the Escrow contract and effectively paralyze governance.
+
+**Possible Mitigation:**
+
+If the Lido DAO acts in good faith, the Tiebreaker Committee can execute a proposal to remove malicious oracle members, thereby restoring the withdrawal finalization process.
+
+If the DAO itself is hostile, the only remaining solution will be the implementation of fully permissionless oracles or withdrawal processes. Early efforts toward this goal are already underway, including [negative rebase sanity checks with a second opinion](https://github.com/lidofinance/lido-improvement-proposals/blob/develop/LIPS/lip-23.md) and [triggerable exits](https://eips.ethereum.org/EIPS/eip-7002).
+
+
+## 2. L2 & Side-Chains Bridges Pause
+
+The wstETH token is widely available on [many L2s and side-chains] (https://docs.lido.fi/deployed-contracts/#lido-multichain).
```

Comment:
The rendering of links is broken because of the space between the link and the caption